### ISSUE TITLE
Build of version 10.0.7 fail cause parent pom io.confluent:common[7.2.0, 7.3.0) is missing

### ISSUE THREAD

**Issue by mbrembilla** [confluentinc/kafka-connect-storage-cloud#511]
Hi all,
I cloned the repo, checked out tag 10.0.7 and ran the maven build, but it failed.
Below you can see the error
```
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR] The project io.confluent:kafka-connect-storage-cloud:10.0.7 has 1 error
[ERROR] Non-resolvable parent POM io.confluent:common:[7.2.0, 7.3.0) for io.confluent:kafka-connect-storage-common-parent:11.0.4: No versions matched the requested parent version range '[7.2.0, 7.3.0)' @ io.confluent:kafka-connect-storage-common-parent:11.0.4, \.m2\repository\io\confluent\kafka-connect-storage-common-parent\11.0.4\kafka-connect-storage-common-parent-11.0.4.pom, line 21, column 13 -> [Help 2]
```
As the error message says, the pom of kafka-connect-storage-common-parent requires the parent pom io.confluent:common at version [7.2.0, 7.3.0), but it's not available in the public repositories.
I searched at:
 - https://mvnrepository.com/artifact/io.confluent/common?repo=confluent-packages
 - http://packages.confluent.io/maven/io/confluent/common/

Why is this repo referencing a version of a parent project that is not yet released? Is it an intended behaviour?

Have a nice day
Thanks

Mauro

**Comment by terrysiu-pm** [comment]
+++1.  Unable to build the project successfully even after manually updating the `io.confluent:common` reference to point to 7.1.1 which is the latest version available in the public repo  😢  I run in to these errors after the manual update:

```
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[WARNING] 'dependencyManagement.dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: com.fasterxml.jackson.core:jackson-databind:jar -> duplicate declaration of version ${jackson.databind.version} @ io.confluent:kafka-connect-storage-common-parent:11.0.4, /home/tsiu/.m2/repository/io/confluent/kafka-connect-storage-common-parent/11.0.4/kafka-connect-storage-common-parent-11.0.4.pom, line 218, column 25
[WARNING] 'dependencyManagement.dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: com.google.protobuf:protobuf-java:jar -> duplicate declaration of version ${protobuf.version} @ io.confluent:kafka-connect-storage-common-parent:11.0.4, /home/tsiu/.m2/repository/io/confluent/kafka-connect-storage-common-parent/11.0.4/kafka-connect-storage-common-parent-11.0.4.pom, line 268, column 25
[ERROR] 'dependencies.dependency.version' for org.junit.jupiter:junit-jupiter-engine:jar is missing. @ line 214, column 21
 @
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]
[ERROR]   The project io.confluent:kafka-connect-storage-cloud:10.1.0-SNAPSHOT (/home/tsiu/code/confluent/kafka-connect-storage-cloud/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for org.junit.jupiter:junit-jupiter-engine:jar is missing. @ line 214, column 21
```

**Comment by terrysiu-pm** [comment]
FWIW, I was able to get `v10.0.6` build to complete, so could be a temporary stopgap until the later release builds are fixed. I tried cloning the upstream projects used by `v10.0.7` and those failed as well so 🤷‍♂️

**Comment by sudeshwasnik** [comment]
@mbrembilla i see. There was another bug due to which 10.0.7 build failed - which was fixed by this -https://github.com/confluentinc/kafka-connect-storage-cloud/pull/513 , and 10.0.8 was released. I mistakenly assumed this issue relates to the same. 
Also because 10.0.8 builds fine in my setup. Because i have access to internal rc confluent/common 7.2.x artifacts. 
I have merged a followup fix https://github.com/confluentinc/kafka-connect-storage-cloud/pull/524. Will release 10.0.9 asap.
